.github/workflows/create-release-branch.yml

```
@@ -0,0 +1,40 @@
+name: Create Release Branch
+# Create a release branch from the current release branch ref 
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+      - name: Create Branch
+        run: |
+          git config --global user.email "ElementorBot@elementor.com"
+          git config --global user.name "ElementorBot"
```

- Move to before the commit action.
- 
```suggestion
          git config --global user.name "${{ secrets.MAINTAIN_USERNAME }}"
          git config --global user.email "${{ secrets.MAINTAIN_EMAIL }}"
```

---

Done